priiloader/include/aes.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
```

Comment:
maybe add a define check to see what libogc version its build with and use libogc if its not the current released version?

in libversion.h are a few defines
```
#define _V_MAJOR_	2
#define _V_MINOR_	3
#define _V_PATCH_	1
```